fritzconnection/lib/fritzhomeauto.py

```diff
@@ -395,7 +395,14 @@ def extract_basicdevicestats_response(response: dict) -> dict:
                 if key == "datatime":
                     value = datetime.datetime.fromtimestamp(value)  # type: ignore
                 content[key] = value
-            content["data"] = list(map(int, stats.text.split(",")))  # type: ignore
+            content["data"] = []
+            for d in stats.text.split(","):
+                try:
+                    content["data"].append(int(d))
+                except:
```

Comment:
No "naked" excepts please. In this case please catch the according "ValueError".